pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/player/compose/ErrorView.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.demo.tv.player.compose
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.media3.common.PlaybackException
+import androidx.tv.material3.ExperimentalTvMaterial3Api
+import androidx.tv.material3.Text
+import ch.srgssr.pillarbox.core.business.SRGErrorMessageProvider
+
+/**
+ * Player error
+ *
+ * @param playerError The player error.
+ * @param modifier The modifier to layout the view.
+ * @param onRetry Action to retry.
+ * @receiver
+ */
+@OptIn(ExperimentalTvMaterial3Api::class)
+@Composable
+fun PlayerError(playerError: PlaybackException, modifier: Modifier = Modifier, onRetry: () -> Unit) {
+    val context = LocalContext.current
+    val errorMessageProvider = remember(context) {
+        SRGErrorMessageProvider(context)
+    }
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .clickable { onRetry.invoke() }
+    ) {
+        Column(modifier = Modifier.align(Alignment.Center), horizontalAlignment = Alignment.CenterHorizontally) {
+            Text(
+                text = errorMessageProvider.getErrorMessage(playerError).second,
+                color = Color.White
+            )
+            Text(text = "Click to retry!", color = Color.LightGray, fontStyle = FontStyle.Italic)
+        }
+    }
```

Comment:
The `Box` is not needed here. You can achieve the same result with the `Column` only:
```suggestion
    Column(
        modifier = modifier.clickable { onRetry() },
        verticalArrangement = Arrangement.Center,
        horizontalAlignment = Alignment.CenterHorizontally
    ) {
        Text(
            text = errorMessageProvider.getErrorMessage(playerError).second,
            color = Color.White
        )
        Text(text = "Click to retry!", color = Color.LightGray, fontStyle = FontStyle.Italic)
    }
```